lib/src/page_list_viewport_gestures.dart

```diff
@@ -242,6 +254,9 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
   void _startMomentum() {
     PageListViewportLogs.pagesListGestures.fine(() => "Starting momentum...");
     final velocity = _panAndScaleVelocityTracker.velocity;
+    final ballisticSimulationInitialVelocityMultiplier =
```

Comment:
What's the reason for creating intermediate variables for these constants? Can you just use them directly down below?

Reply:
It would be a very long line + This is how the code was formatted before me

Reply:
It sounds like the answer is that yes you can use them directly, and there's no technical reason not to. With that being the case, please remove the intermediate variables and use them directly down below.